10/umbraco-cms/reference/configuration/maximumuploadsizesettings.md

```diff
@@ -50,7 +50,7 @@ An example of a configuration could look something like this:
 }
 ```
 
-`MaxRequestLength` is specified in kilobytes, so this configuration would limit requests, and therefore uploaded files, to 2 megabytes, and a maximum query string length of 90 characters.
+`MaxRequestLength` is specified in kilobytes. This configuration would limit requests, and therefore uploaded files, to 2 megabytes, and a maximum query string length of 90 characters.
```

Comment:
Does it make sense to say "This configuration **will** limit requests ... ", instead?
At least that sounds a little better to me 🤔 What do you think?

Reply:
@sofietoft Definitely agree!